build/rpm/dependencies.yaml

```diff
@@ -0,0 +1,19 @@
+# This file holds a list of RPM dependencies that apply to Puppet module RPMs.
+#
+# This is specifically needed since we do not want to fork any external modules
+# if at all possible.
+#
+# Requirement versions are pulled directly from the metadata.json file in the module
+---
+'grafana':
+  :obsoletes:
+    'pupmod-bfraser-grafana': '2.5.0-2016.1'
+  :requires:
+    - 'pupmod-puppetlabs-stdlib'
+'elasticsearch':
+  :obsoletes:
+    'pupmod-elasticsearch-grafana': '0.11.0-2016.1'
+  :requires:
+    - 'pupmod-richardc-datacat'
+    - 'pupmod-puppetlabs-java'
+    - 'pupmod-puppetlabs-stdlib'
```

Comment:
elasticsearch requires puppet/yum if you want to pin the version of elasticsearch rpm